js/packages/common/src/utils/ids.ts

```diff
@@ -78,7 +78,11 @@ export const METAPLEX_ID =
   'p1exdMJcjVao65QdewkaZRUnU6VPSXhus9n2GzWfh98' as StringPublicKey;
 
 export const PACK_CREATE_ID = new PublicKey(
-  'packFeFNZzMfD9aVWL7QbGz1WcU7R9zpf6pvNsw2BLu',
+  'GTFVL9hA9AhWjNxhH99Hw2mp1MycFqRTjDS4xAHREheC',
```

Comment:
@austbot We need to deploy the latest codebase to packFeFNZzMfD9aVWL7QbGz1WcU7R9zpf6pvNsw2BLu, so I can change the program id here